libs/SmartStore/SmartStore/Classes/SFSmartStoreInspectorViewController.h

```diff
@@ -32,6 +32,10 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface SFSmartStoreInspectorViewController : UIViewController <UICollectionViewDataSource, UICollectionViewDelegateFlowLayout, UITextViewDelegate>
 
+/**
+ Constructor.
+ */
+- (instancetype) init;
```

Comment:
New constructor - will show default smartstore (of user if there is one or global otherwise).

Reply:
It's also the behavior on Android (on Android you pass an extras bundle when starting the activity but when it's null we show the default store).